### ISSUE TITLE
ImportError: cannot import name '_gi'

### ISSUE THREAD

**Issue by junior466** [mvo5/unattended-upgrades#234]
I recently upgraded Python on my Ubuntu 16.04 system and noticed that my unattended upgrades no longer work. I am receiving the following error:

    ImportError: cannot import name '_gi'

My python versions:

    python -V
    Python 2.7.12

    python -V
    Python 3.6.9

I've tried installing Python Gi but it just says its already installed. 

Any idea how I can correct this issue?

**Comment by rbalint** [comment]
This must be a typo:
```
python -V
Python 3.6.9
```

16.04 ships Python 3.5 (you have 3.6 installed either locally or from a PPA), but python3-gi is built only for 3.5:
```
root@x:~# dpkg -L python3-gi | grep x86
/usr/lib/python3/dist-packages/gi/_gi.cpython-35m-x86_64-linux-gnu.so
```

 You can revert the default Python version to the one shipped by 16.04.

**Comment by junior466** [comment]
@rbalint Its definitely not a typo but a beginner mistake. I needed to upgrade python and somehow ended up with 3.6 as well. I would love to revert it to 3.5 and get my system back to normal without a complete reinstall but I don't know where to begin. The only issue I see so far is unattended upgrades not working. 

Thanks for your help.
